templates/ansible/resource.erb

```diff
@@ -274,15 +274,39 @@ def resource_to_request(module):
             return_vals[k] = v
 
     return return_vals
+<% unless false?(object.unwrap_resource) -%>
```

Comment:
I'm going to go ahead and trust your judgment here but to my still-new-at-ruby-eye, this is less readable than `if object.unwrap_resource`

Reply:
This is something we should bring up for a potential bug fix-it.

In Ruby, we have true, false, and nil.

nil == false, but false != nil.

In this case, we want true false, but we don't want to have nils confused in it as well. That's what we made the false? function for.

I'm not a huge fan of this. I know the Ruby community must have a better way to fix this.